customer-config/inactivity-logout/iam.inactivity-logout.v1.json

```diff
@@ -3,7 +3,13 @@
   "$schema": "https://json-schema.org/draft/2020-12/schema",
   "title": "Inactivity-logout",
   "description": "A schema for ui inactivity logout kind for Customer Controlled Config service.",
-  "type": "number",
-  "default": 2880,
-  "minimum": 5
+  "type": "object",
+  "properties": {
+    "minutes": {
```

Comment:
isn't it a required property? by your schema you can omit the minutes, but that is not correct i guess...?